Dockerfiles/pcluster-amazonlinux-2/Dockerfile

```diff
@@ -128,6 +128,20 @@ RUN mkdir -p $(dirname "${SPACK_ROOT}") \
     && spack buildcache create -a -u /bootstrap-gcc-cache $(spack find --format '/{hash}') \
     && rm -rf $(dirname "${SPACK_ROOT}") /root/.spack
 
+# Sign the buildcache
+RUN --mount=type=secret,id=bootstrap_gcc_key \
+    mkdir -p $(dirname "${SPACK_ROOT}") \
+    && git clone https://github.com/spack/spack "${SPACK_ROOT}" \
+    && pushd "${SPACK_ROOT}" && git checkout ${SPACK_COMMIT} && popd \
+    && . "${SPACK_ROOT}/share/spack/setup-env.sh" \
+    && wc /run/secrets/bootstrap_gcc_key \
```

Comment:
Why are we running `wc` on the secret?

Reply:
Thansk for catching. That's a leftover from the debugging. I'll remove it.